chessmate/engines.py

```diff
@@ -169,7 +169,57 @@ def evaluate(self, board: chess.Board) -> None:
             self.evaluation_function.evaluate(board)
         )
 
-
+class PrioritizeBishopMoves(Random):
+    """Engine that only moves bishops when an option.
+    Default to random engines if no bishop moves available"""
+    
+    def __init__(self):
+        """See parent docstring"""
+        super().__init__()
+        self.name = "Prioritize Bishop Moves"
+        
+    def evaluate(self, board: chess.Board) -> None:
+        """Assigns same value to each move since eventually going to choose random move """
+        self.reset_move_variables()
+        
+        possible_moves = list(board.legal_moves)
+        for m in possible_moves:
+            if get_piece_at(board, str(m)[:2].upper()) == "B":
+                self.legal_moves[m] = 1
+        
+        #If no bishop moves available, all moves are same priority
+        
+        if not self.legal_moves:
+            self.legal_moves = {
+                possible_moves[i]: 1 for i in range(len(possible_moves))
+            }
+        self.material_difference.append(
+            self.evaluantion_function.evaluate(board)
+        )
+class PrioritizeKnightMoves(Random):
+    """Like PrioritizeBishopMoves, but for knights"""
```

Comment:
Can you write the docstring such that it can be understood without  referring to ```PrioritizeBishopMoves```? Additionally, can you make sure there's a space between the """ and the first & last characters of the docstring for readability and consistency?